examples/complex/package.json

```diff
@@ -18,9 +19,10 @@
   "license": "ISC",
   "dependencies": {
     "minimist": "^1.2.0",
+    "openzeppelin-solidity": "~1.9.0",
     "openzeppelin-zos": "^1.9.0-beta",
     "truffle": "^4.1.5",
-    "zos-lib": "file:../../"
```

Comment:
Out of curiosity: is there a difference between having or not the trailing slash?

Reply:
It was automatically done by npm. :man_shrugging: I figure this was the canonical thing so I left it.